### DISCUSSION TITLE
I need some ideas on the following error

### DISCUSSION THREAD

**Discussion by jzs11** [vuejs/language-tools#796] [Q&A] [answered]
Hi chaps,

Need some help on the following error, recently I encountered. 
The "@" as the root path for my typescript files seem has not been picked up by the VS Code.

![image](https://user-images.githubusercontent.com/2770401/146334284-0c1f8e33-f471-4571-9429-c9a2dce41c57.png)

my tsconfig.json has the following setting:
![image](https://user-images.githubusercontent.com/2770401/146334423-1060b705-3945-425e-a3a2-d12ad3ac33fb.png)

Just happened today...

Thanks,
Jing

**Comment by jzs11** [comment]
I can confirm everything is working fine in my Win11 machine, but not on 

System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 972.67 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.2.0 - /usr/local/bin/node
    npm: 8.2.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Safari: 15.2
  npmPackages:
    @vitejs/plugin-vue: 2.0.1 => 2.0.1 
    vite: 2.6.14 => 2.6.14

**Comment by Shinigami92** [comment]
https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping

Did you also set `baseUrl`?

**Reply by jzs11** [reply]
I have setup the baseUrl.

I can give you access to my repo as well, to help me figure out that 2.7.0+ vite production mode.
However it requires you to install dotnet sdk 6 and mongodb etc...

Do you want to give a go? Let me know.

**Comment by johnsoncodehk** [comment]
Could you provide a minimal reproduction?

**Reply by jzs11** [reply]
Hey dude, invited you into the my repo. Give a go. Let me know your thought. The UI source code is under WebUI/clientapp

**Comment by jzs11** [comment, marked as answer]
I found out the problem, it is an Casing issue in my iOS file path
![image](https://user-images.githubusercontent.com/2770401/146472400-0885d6ef-0341-4d7a-bab9-dc907e3b2ba6.png)

After I change my folder name from Systemis to systemis, everything works fine.... Thank you guys!!
